### ISSUE TITLE
Add wikitext for default highligt: @@highlight@@

### ISSUE THREAD

**Issue by twMat** [Jermolene/TiddlyWiki5#2228]
`@@background:yellow; content@@` works but there is no default option without parameters.

I propose `@@content@@` as a quick way to get the typical faint(!) yellow background on the content.

**Comment by tobibeer** [comment]
I'd agree that such a basic highlight is useful.

**Comment by twMat** [comment]
bump

**Comment by linonetwo** [comment]
So you mean transform `@@background:yellow; content@@` to a `span` tag, and `@@content@@` to `mark` tag.

**Comment by pmario** [comment]
@linonetwo ... `@@` is about [Styles and classes](https://tiddlywiki.com/#Styles%20and%20Classes%20in%20WikiText) ... imo not the same as `<mark>`

**Comment by linonetwo** [comment]
You are right, `@@` only add style or class to the outer element, not adding a warper element.

Maybe `@@highlight@@ ` can add a special highlight class, that can be config in the palette.

**Comment by pmario** [comment]
I think the OT is actionable and imo makes sense, so I'll bump the issue

**Comment by Jermolene** [comment]
It's a good idea. I don't think there are any backwards compatibility considerations because `@@foobar@@` at present doesn't do anything useful.

**Comment by twMat** [comment]
So no misunderstanding: I'm hoping it will work also for `@@strings with spaces@@`

**Comment by pmario** [comment]
>So no misunderstanding: I'm hoping it will work also for @@strings with spaces@@

Yes is will. It's like any other "emphasis" rule we have eg: bold

**Comment by pmario** [comment]
@twMat -- IMO this one can be closed -- The default colour has been implemented some time ago. See: https://tiddlywiki.com/#Formatting%20in%20WikiText

**Comment by linonetwo** [comment]
I see, it generates `<span class="tc-inline-style">highlight</span>` where `tc-inline-style` can be moded.
